frontend/src/pages/ClientVenues.jsx

```diff
@@ -1,8 +1,55 @@
+import React from "react";
+import Navbar from "../components/Navbar";
+import Saddledome from "../assets/Saddle.jpg"
+import TMobile from "../assets/T-Mobile2.png"
+import Amalie from "../assets/Amalie.jpg"
+
+const BUTTON_CONTAINER = "bg-primary w-64 flex flex-col items-center text-xl my-8 mx-auto py-2  rounded-lg hover:text-dark ease-in duration-300 font-bold"
+
+
 function ClientVenues() {
 	return ( 
-		<div className="bg-dark min-h-screen pt-36 text-white">
-			<h1>client venues</h1>
-		</div>
+		<>
+			<Navbar />
```

Comment:
You can take navbar out of client Venus because I already declared it in the app so you don't have to put it on every page